cg/cli/upload/gisaid.py

```diff
@@ -0,0 +1,41 @@
+"""Code for uploading genotype data via CLI"""
+import logging
+from typing import List
+
+import click
+from cg.store import models
+
+from cg.meta.upload.gisaid import UploadGisaidAPI
+
+from .utils import suggest_cases_to_upload
+from cg.meta.meta import MetaAPI
+
+LOG = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument("family_id", required=True)
+@click.pass_context
+def gisaid(context, family_id):
+    """Upload mutant analysis data to GISAID."""
+
+    meta_api = MetaAPI(context.obj)
+    status_db = meta_api.status_db
+    click.echo(click.style("----------------- GISAID -------------------"))
+
+    case_object = status_db.family(family_id)
+    ticket_number = case_object.name
+    samples: List[models.Sample] = status_db.Sample.query.filter_by(
```

Comment:
Please use the functions that exist in the API (the status_db api) or add the functionality there if it does not exist. Code like this will be hard to hunt down and change when we change the API

Reply:
That was the plan 👍